src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,39 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  #this;
+  constructor(name, health, hidden = false) {
```

Comment:
The 'health' parameter should not be in the constructor of the 'Herbivore' class as the health is meant to default to 100 and is not supposed to be set during the creation of 'Herbivore' instances according to the task description.